### PR TITLE
Feature/naming cleanup

### PR DESCRIPTION
**Pull request by mprorock**
fixes issues related to issues: 

- #41 NPM Scoping: `@transmute` to `@w3c-ccg` and references in package.json to `file:packages/...` as appropriate for dev
- #42 Several dependency version updates for security reasons
- #43 Update contributors page on the docs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,5 +29,10 @@
     "lerna": "^3.20.2",
     "prettier": "^2.0.5",
     "typescript": "^3.8.3"
+  },
+  "dependencies": {
+    "@transmute/did-key-ed25519": "^0.2.1-unstable.24",
+    "@transmute/did-key-x25519": "^0.2.1-unstable.24",
+    "@w3c-ccg/universal-wallet": "file:packages/universal-wallet"
```
Comment (OR13) [inline package.json:36]: this is not necessary, you can link packages of the same version with symlink using `lerna link`

**Comment by OR13** [comment]
https://www.npmjs.com/package/@w3c-ccg <- this namespace does not exist (yet), would need to have it be created and setup to accept this PR.

File: package.json (hunk 1 above)
Comment (mprorock) [reply package.json:36]: not a bad idea - will update to lerna

**Review by OR13 — CHANGES_REQUESTED** [review]
1. Revert the namespace changes until we confirm with chairs / decide how to handle that.
2. Revert the breaking changes to `lerna`

**Comment by mprorock** [comment]
holding on this for another day or so while we wait for chair input.  will make adjustments as request by @OR13 to account for `lerna`

**Comment by OR13** [comment]
@mprorock I will close this PR in 1 week, if the code changes are not reverted :)


Suggest we split this up into phases... also regarding npm namespaces, there is nothing stopping you from doing `@mesur/universal-wallet-did-web-plugin` and committing the source to this repo as I have done for transmute, while we wait for guidance from the CCG.... 

Alternatively, as an owner the `@transmute/universal-wallet-edv-plugin` I can give you / this repo publish token for the npm module and setup CI/CD with or without the chair input (they can't see tokens)... we do something similar here: https://material-did.com/?path=/story/*

probably CI/CD with automatic builds will make all downstream consumers happy and the NPM namespace is mostly irrelevant to setting that up.

**Comment by mprorock** [comment]
> @mprorock I will close this PR in 1 week, if the code changes are not reverted :)
> 
> Suggest we split this up into phases... also regarding npm namespaces, there is nothing stopping you from doing `@mesur/universal-wallet-did-web-plugin` and committing the source to this repo as I have done for transmute, while we wait for guidance from the CCG....
Agreed - will go through and break this up towards the end of the week

**Comment by wyc** [comment]
Thanks for the interest and suggested of a CCG npm package repo, please see here for an update: https://github.com/w3c-ccg/vc-http-api/issues/79#issuecomment-737597078

**Comment by mprorock** [comment]
updated with remove of `@w3c-ccg` namespacing for now while we sort that out @OR13

File: package.json [hunk 2]
```diff
@@ -29,5 +29,11 @@
     "lerna": "^3.20.2",
     "prettier": "^2.0.5",
     "typescript": "^3.8.3"
+  },
+  "dependencies": {
+    "@transmute/did-key-ed25519": "^0.2.1-unstable.24",
+    "@transmute/did-key-x25519": "^0.2.1-unstable.24",
+    "@w3c-ccg/universal-wallet": "file:packages/universal-wallet",
```
Comment (OR13) [inline package.json:36]:
```suggestion
    "@transmute/universal-wallet": "^0.4.1",
```

Comment (OR13) [reply package.json:36]: This is required to fix lerna issues in this PR.

Comment (OR13) [reply package.json:36]: lerna automatically handles linking, so the `file:` is not needed, and breaks things.

**Review by OR13 — CHANGES_REQUESTED** [review]
fix lerna issue

File: package.json (hunk 2 above)
Comment (mprorock) [reply package.json]: updated in commit 733b20921295b8f495ff0ad1776a9aeb698b521b

**Review by OR13 — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Pull request merged**